docs/install/installing-istio.md

```diff
@@ -261,6 +261,24 @@ kubectl apply -f https://raw.githubusercontent.com/knative/serving/master/third_
 **Note:** This method is only for development purposes. The production readiness of the above 
 installation method is not ensured. For a production-ready installation, see the `helm` installation method above.
 
+If you follow either of the above steps, your service and deployment for the local gateway are both named `cluster-local-gateway`,
+and you do not need to update gateway configmap `config-istio` under `knative-serving` namespace, because Knative Serving
+can by default use the local gateway with the name `cluster-local-gateway`.
+
+However, if you create custom service and deployment for local gateway with a name other than `cluster-local-gateway`, you
```

Comment:
Maybe a new heading here to call attention to the user story / task?
e.g. Updating the `config-istio` configmap to use a non-default local gateway

@samodell wdyt?

Reply:
I'd remove "However" here